script.js

```diff
@@ -0,0 +1,108 @@
+const hamburger = document.querySelector('.mobile-navbar');
+const openMenuContainer = document.querySelector('article.mobile-menu');
+const closeMenuBtn = document.querySelector('.close-menu-btn');
+
+hamburger.addEventListener('click', () => {
+  openMenuContainer.classList.toggle('d-none');
+});
+
+closeMenuBtn.addEventListener('click', () => {
+  openMenuContainer.classList.toggle('d-none');
+});
+
+const featuredSpeakers = [
+  {
+    id: 1,
+    fullName: 'Jane Cooper',
+    image: './media/images/speakers/featured-speaker-1.jpg',
+    alt: 'speaker-1',
+    title: 'Co-founder at Barone LLC.',
+    address: '2118 Thornridge Cir. Syracuse, Connecticut 35624',
+  },
+  {
+    id: 2,
+    fullName: 'Wade Warren',
+    image: './media/images/speakers/featured-speaker-2.jpg',
+    alt: 'speaker-2',
+    title: 'CTO at Acme Co.',
+    address: '2972 Westheimer Rd. Santa Ana, Illinois 85486 ',
+  },
+  {
+    id: 3,
+    fullName: 'Cameron Williamson',
+    image: './media/images/speakers/featured-speaker-3.jpg',
+    alt: 'speaker-3',
+    title: 'CEO at Biffco Enterprises Ltd.',
+    address: '2118 Thornridge Cir. Syracuse, Connecticut 35624',
+  },
+  {
+    id: 4,
+    fullName: 'Leslie Alexander',
+    image: './media/images/speakers/featured-speaker-4.jpg',
+    alt: 'speaker-4',
+    title: 'Lead SWE at Binford Ltd.',
+    address: '2972 Westheimer Rd. Santa Ana, Illinois 85486',
+  },
+  {
+    id: 5,
+    fullName: 'Robert Fox',
+    image: './media/images/speakers/featured-speaker-5.jpg',
+    alt: 'speaker-5',
+    title: 'Senior SWE at Abstergo Ltd.',
+    address: '8502 Preston Rd. Inglewood, Maine 98380',
+  },
+  {
+    id: 6,
+    fullName: 'Jacob Jones',
+    image: './media/images/speakers/featured-speaker-6.jpg',
+    alt: 'speaker-6',
+    title: 'Associate SWE at Big Kahuna Burger Ltd.',
+    address: '3517 W. Gray St. Utica, Pennsylvania 57867',
+  },
+];
```

Comment:
- [ ] Nice, you have followed the requirements here to call the image from the Js file, however, let us fix this issue by attaching the background as given in our design.

-----------------------------------------------Original------------------------------------------------

![navbar4](https://user-images.githubusercontent.com/55635977/128439119-1c413a8a-0a4b-41ab-b249-945c3fcdbf9c.png)


-----------------------------------------------Your------------------------------------------------

![navbar3](https://user-images.githubusercontent.com/55635977/128439110-21986473-0b62-4379-8303-0830cf653d61.png)

Reply:
Well you tried to work on this but you can still improve it in the future to look exactly like the design!